godtools/Views/TractElements/Paragraph.swift

```diff
@@ -27,18 +27,21 @@ class Paragraph: BaseTractElement {
         self.frame = buildFrame()
     }
     
-    override func textStyle() -> (style: String, width: CGFloat, height: CGFloat, alignment: NSTextAlignment, xMargin: CGFloat, yMargin: CGFloat) {
+    override func textStyle() -> TextStyle {
+        let textStyle = super.textStyle()
+        
         var xMargin = BaseTractElement.xMargin
         if BaseTractElement.isCardElement(self) {
             xMargin = Card.xPaddingConstant
         }
         
-        return ("toolFrontSubTitle",
-                self.width,
-                0.0,
-                NSTextAlignment.left,
-                xMargin,
-                BaseTractElement.yMargin)
+        textStyle.style = "toolFrontSubTitle"
+        textStyle.width = self.width
+        textStyle.alignment = .left
```

Comment:
How will the operating system handle this for right to left languages (Arabic, Hebrew and others)? We want to make sure we support those well for this new version of God Tools.